ksqldb-parser/src/main/java/io/confluent/ksql/parser/ParsingException.java

```diff
@@ -18,31 +18,29 @@
 import static java.lang.String.format;
 
 import java.util.Optional;
-import org.antlr.v4.runtime.RecognitionException;
 
 public class ParsingException
     extends RuntimeException {
 
   private final int line;
   private final int charPositionInLine;
+  private final String unloggedDetails;
 
   public ParsingException(final String message, final Optional<NodeLocation> nodeLocation) {
     this(
         message,
-        null,
         nodeLocation.map(NodeLocation::getStartLineNumber).orElse(1),
         nodeLocation.map(NodeLocation::getStartColumnNumber).orElse(0)
     );
   }
 
   public ParsingException(
       final String message,
-      final RecognitionException cause,
```

Comment:
Dropped this cause from the stacktrace. It doesn't provide necessary debugging information, and we can't control what the parser will include in the message.